nss/src/passwd/mod.rs

```diff
@@ -108,6 +108,12 @@ fn get_entry_by_name(name: String) -> Response<Passwd> {
             }
         };
 
+        // This is a fake call done by PAM to avoid attacks, so we need to special case it to avoid spamming
+        // logs with "Not Found" messages, as this call is done quite frequently.
+        if name.contains("pam_unix_non_existent") {
```

Comment:
You can probably just use:

```suggestion
        if name == "pam_unix_non_existent" {
```

And maybe reference the code doing it: https://github.com/linux-pam/linux-pam/blob/8401cef10cd5f62849c5fcfef4c82db92712296c/modules/pam_unix/support.c#L700

Reply:
Yeah, I thought about using direct comparison, but then wasn't sure whether there could be other formats used by PAM to achieve the same thing. Overall, I think it's safe to keep it as  `.contains()` rather than a direct comparison. Do you think there could be any issues?

Reply:
No, not an issue... I don't think that code will ever change TBH, and in this kind of things the more restrict we are, the best.

But it's not a strong opinion.

Reply:
Nice!